tests/components/RxView/rx-view.test.js

```diff
@@ -117,12 +117,6 @@ describe('RxView component', () => {
     jest.resetModules();
   });
 
-  it('contains relevant patient information if patient is in context', () => {
-    setup(patient, medListPhase, medications, prescription);
-    expect(renderedComponent.find('.patient-data-text').text()).toContain(patient.name);
-    expect(renderedComponent.find('.patient-data-text').text()).toContain(patient.id);
-  });
-
   it('has default view elements', () => {
```

Comment:
With the above test removed, can you update this test to assert that the patient banner is included?

Reply:
No problem.